source/common/network/address_impl.h

```diff
@@ -54,23 +60,23 @@ class Ipv4Instance : public InstanceBase {
   /**
    * Construct from an existing unix IPv4 socket address (IP v4 address and port).
    */
-  explicit Ipv4Instance(const sockaddr_in* address);
+  explicit Ipv4Instance(const sockaddr_in* address, const std::string& sock_interface = "");
```

Comment:
Nit: `absl::string_view` for all `const std::string&`s.

Reply:
Done!